app/views/provider/admin/onboarding/wizard/backend_api/new.html.slim

```diff
@@ -25,4 +25,4 @@ ol.explain.fa-ul
   button type="submit" class="button button--next"
     = 'Add this Backend'
 
-= javascript_pack_tag 'apiap_wizard_backend_api'
+= javascript_pack_tag 'wizard_backend_api'
```

Comment:
Well I think you need to change the filename as well

Reply:
It is changed 
<img width="1044" alt="Screenshot 2021-07-12 at 15 38 55" src="https://user-images.githubusercontent.com/11672286/125297181-4f623d00-e327-11eb-9023-d85b92ac1fd5.png">